src/scss/layout/_footer.scss

```diff
@@ -0,0 +1,41 @@
+.footer {
+  color: var(--white);
+  background-color: var(--navy-blue);
+  padding: 40px 15px 72px;
+
+  &__title {
+    margin-bottom: 16px;
+  }
+
+  &__links {
+    margin-bottom: 12px;
+  }
+
+  &__link {
+    margin-bottom: 16px;
+
+    .link {
```

Comment:
This is missing the line:       color: var(--white);  to change the color from "electric-blue" to white.